### PR TITLE
Migration androidx

### PR DESCRIPTION
**Pull request by cdsap**
Current dependencies:
 
        androidx       : '1.0.0-rc01'
        espresso       : '3.1.0-alpha4'
        espresso_rules : '1.0.0-alpha4'
        espresso_runner: '1.0.0-alpha4'

Also includes:

- Gradle update to 3.14/4.6

- Target SDK 28(aapt problems)

- `getAdapter()` includes `Nullable` annotation on Recycler androidx. Check changes on `Actions.kt`

We need to discuss if we want to offer a different artifact as the request on the issues.

### CODE REVIEW

**Comment by cdsap** [comment]
CI failing on:
`Could not determine artifacts for androidx.test.espresso:espresso-contrib:3.1.0-alpha4: Skipped due to earlier error`
Locally OK. Tomorrow I'll check it

File: buildsystem/dependencies.gradle [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
         maven          : '1.5',
         oss            : '4.5.4',
-        support_library: '27.1.1',
-        espresso       : '3.0.1',
-        espresso_rules : '1.0.1',
-        espresso_runner: '1.0.1',
+        androidx       : '1.0.0-rc01',
+        espresso       : '3.1.0-alpha4',
+        espresso_rules : '1.0.0-alpha4',
+        espresso_runner: '1.0.0-alpha4',
```
Comment (atetc) [inline buildsystem/dependencies.gradle]: Same version, you can use 1 var

File: buildsystem/dependencies.gradle [hunk 2]
```diff
@@ -20,16 +20,16 @@ ext.libraries = [
...
-        annotations     : "com.android.support:support-annotations:$versions.support_library",
-        espresso_runner : "com.android.support.test:runner:$versions.espresso_runner",
-        espresso_rules  : "com.android.support.test:rules:$versions.espresso_rules",
-        espresso_core   : "com.android.support.test.espresso:espresso-core:$versions.espresso",
-        espresso_web    : "com.android.support.test.espresso:espresso-web:$versions.espresso",
-        espresso_contrib: "com.android.support.test.espresso:espresso-contrib:$versions.espresso",
-        espresso_intents: "com.android.support.test.espresso:espresso-intents:$versions.espresso",
+        appcompat       : "androidx.appcompat:appcompat:$versions.androidx",
+        recyclerview    : "androidx.recyclerview:recyclerview:$versions.androidx",
+        design          : "com.google.android.material:material:$versions.androidx",
```
Comment (atetc) [inline buildsystem/dependencies.gradle]: material now

Comment (nekdenis) [reply buildsystem/dependencies.gradle]: @atetc https://github.com/agoda-com/Kakao/pull/112 - all checks passed

**Comment by atetc** [comment]
I am with @nekdenis let's use #112

**Comment by cdsap** [comment]
sure, I close this PR

**Pull request closed without merging**